### PR TITLE
Misc: remove unneeded file and clean up a 32bit check

### PR DESCRIPTION
**Pull request by rayanHappy**
### Description of Changes

As the title says.
### Rationale behind Changes

less stuff.
### Suggested Testing Steps

tested, things should work but you can build it if you want.

### CODE REVIEW

File: cmake/SearchForStuff.cmake [hunk 1]
```diff
@@ -102,18 +102,7 @@ if(ENABLE_TESTS)
...
-#----------------------------------------
-include(ApiValidation)
-
-# Blacklist bad GCC
-if(GCC_VERSION VERSION_EQUAL "7.0" OR GCC_VERSION VERSION_EQUAL "7.1")
-	GCC7_BUG()
-endif()
-
-if((GCC_VERSION VERSION_EQUAL "9.0" OR GCC_VERSION VERSION_GREATER "9.0") AND GCC_VERSION LESS "9.2")
+if(GCC_VERSION GREATER_EQUAL "9.0" AND GCC_VERSION LESS "9.2")
```
Comment (TellowKrinkle) [inline cmake/SearchForStuff.cmake:105]:
```suggestion
if(GCC_VERSION VERSION_GREATER_EQUAL "9.0" AND GCC_VERSION VERSION_LESS "9.2")
```

**Comment by TellowKrinkle** [comment]
Is there a reason you closed this?

**Comment by rayanHappy** [comment]
> Is there a reason you closed this?

yeah, i wanted it to be only one commit, didn't know how to fix it so i closed it.

**Comment by kamfretoz** [comment]
You can squash the commit

**Comment by TellowKrinkle** [comment]
Ahh
You can clone locally, then `git rebase -i HEAD~~`, and edit the commits there
Or if you have the "maintainers can edit" checkbox checked, I can do that for you

**Comment by rayanHappy** [comment]
okay, thank you both for the info.

**Pull request closed without merging**